src/main/kotlin/ch/srgssr/pillarbox/monitoring/event/model/EventRequest.kt

```diff
@@ -64,6 +64,15 @@ private class DataDeserializer : JsonDeserializer<Any?>() {
         .build()
   }
 
+  fun isHackerOrRobot(userAgent: UserAgent): Boolean =
+    listOf("DeviceClass", "LayoutEngineClass", "AgentClass", "AgentSecurity")
```

Comment:
Is it possible to use the constant from `UserAgent`?
```suggestion
    listOf(UserAgent.DEVICE_CLASS, UserAgent.LAYOUT_ENGINE_CLASS, UserAgent.AGENT_CLASS, UserAgent.AGENT_SECURITY)
```

Reply:
Done